### PR TITLE
Adding example to remove an image

### PR DESCRIPTION
**Pull request by renuHadke**
This playbook will remove specified image if present.

Signed-off-by: renuhadke <hadke.r@husky.neu.edu>

### CODE REVIEW

File: examples/remove_image.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+#This ansible playbook will remove an image
+
+- hosts: localhost
+  tasks:
+    - name: Test Remove image
```
Comment (rhatdan) [inline examples/remove_image.yml]: I would remove the "Test" and just call this "Remove image"

Comment (renuHadke) [reply examples/remove_image.yml]: Sure, I will make the change

**Comment by Phadated** [comment]
@renuHadke LGTM

**Comment by nehajain1809** [comment]
LGTM

**Pull request merged**